Makefile

```diff
@@ -25,17 +25,18 @@ help: ## Display this help
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[0-9a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
 OLM_PACKAGE_NAME ?= cert-manager
-IMG_BASE ?= gcr.io/jetstack-richard/cert-manager
+IMG_BASE_DEFAULT := ttl.sh/$(shell uuidgen)/cert-manager
+IMG_BASE ?= $(IMG_BASE_DEFAULT)
```

Comment:
Needed `IMG_BASE_DEFAULT` (set using `:=` syntax) so that `IMG_BASE` remains constant across all the rules in the Makefile.
Otherwise  `uuidgen` is executed every time the variable is used.